cscs-checks/apps/vasp/vasp_check.py

```diff
@@ -8,16 +8,19 @@
 
 
 class VASPCheck(rfm.RunOnlyRegressionTest):
-    keep_files = ['OUTCAR']
-    maintainers = ['LM']
-    tags = {'scs'}
-    strict_check = False
+    scale = parameter(['small', 'large'])
+    variant = parameter(['maint', 'prod'])
     modules = ['VASP']
+    executable = 'vasp_std'
+    keep_files = ['OUTCAR']
     extra_resources = {
         'switches': {
             'num_switches': 1
         }
     }
+    strict_check = False
+    maintainers = ['LM']
+    tags = {'scs'}
```

Comment:
Perhaps this tag has no meaning anymore... :-(

Reply:
Tag removed